scripts/check-layout-overrides.sh

```
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# Detects custom layouts that override Congo theme files at the same path.
+# Run after `task update-theme` to find layouts that may need reconciliation.
+
+THEME_DIR="themes/congo/layouts"
+CUSTOM_DIR="layouts"
+
+overrides=()
+
+while IFS= read -r -d '' custom_file; do
+  rel="${custom_file#${CUSTOM_DIR}/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)